### DISCUSSION TITLE
[Announcement] Welcome & Setup

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs524-program-analysis#359] [Announcements]
Hi everyone.

Welcome to CS524. I am looking forward to meeting you in class.
Here are some things you should set up for this course. The details will be introduced in the first lecture. 

- Submit your account via the google form. (TAs will give the link below.)
- Join the [Gradescope](https://www.gradescope.com). The Entry code is `57KJ3G`

Hope you enjoy the class!

**Comment by KAIST-JongchanPark** [comment]
Here is the link for the google form.

https://forms.gle/esqmpgU5zJa4bSGF8
